clean.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd $(dirname "$0")
+
+# clear submodules
+bash ./etc/scripts/get_submodule_paths.sh | xargs rm -rf
```

Comment:
I guess this is fine, bu could also do something like `git clean -dfx .` in each submodule so we don't fetch a fresh copy?